src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,41 +1,153 @@
-import { Story, Meta } from '@storybook/react';
+import type { Meta, Story, StoryObj } from '@storybook/react';
+import { within, userEvent } from '@storybook/testing-library';
+import clsx from 'clsx';
 import React from 'react';
+import Tooltip from './Tooltip';
+import styles from './Tooltip.stories.module.css';
+import { Button, Props } from '../Button/Button';
 
-import { Tooltip, Props } from './Tooltip';
-import { TextPassage } from '../TextPassage/TextPassage';
+/* TODO: replace ForwardRefButton with EDS button since it's already using forwardRef */
+const ForwardRefButton = React.forwardRef((props: Props, ref) => (
+  <Button {...props} buttonRef={ref} />
+));
+ForwardRefButton.displayName = 'ForwardRefButton';
```

Comment:
This is [required](https://github.com/atomiks/tippyjs-react#component-children) for for Tippy. Button that will be imported from EDS already uses [forwardRef](https://github.com/chanzuckerberg/edu-design-system/blob/9aadd55d95623cd0d777c88edcc93bc8e3edbf13/packages/components/src/Button/button.tsx#L34)